src/assets/lang/en/i-have-been-pawned.json

```diff
@@ -28,8 +28,8 @@
       "description": "Your email has been found in ",
       "breaches": " data breaches and ",
       "breach": " data breach and ",
-      "pastes ": " pastes.",
-      "paste": " paste.",
+      "pastes ": "paste.",
```

Comment:
It would be better if the keys and values fit together

Reply:
Done

Reply:
Done